### PR TITLE
Fix `blank line before declarations` rule code example

### PR DESCRIPTION
**Pull request by k-taro56**
## Description

The code example used to demonstrate disallowed code was correctly formatted (i.e., there were blank lines before declarations). This change corrects the example to properly reflect code that violates the rule.

## Checklist

Before submitting the PR, please check following (checks which are not relevant may be ignored):
- [ ] Commit message are well written. In addition to a short title, the commit message also explain why a change is made.
- [ ] At least one commit message contains a reference `Closes #<xxx>` or `Fixes #<xxx>` (replace`<xxx>` with issue number)
- [ ] Tests are added
- [x] KtLint format has been applied on source code itself and violations are fixed
- [x] PR title is short and clear (it is used as description in the release changelog)
- [x] PR description added (background information)

[Documentation](https://pinterest.github.io/ktlint/) is updated. See [difference between snapshot and release documentation](https://github.com/pinterest/ktlint/tree/master/documentation)
- [x] [Snapshot documentation](https://github.com/pinterest/ktlint/tree/master/documentation/snapshot) in case documentation is to be released together with a code change
- [x] [Release documentation](https://github.com/pinterest/ktlint/tree/master/documentation/release-latest) in case documentation is related to a released version of ktlint and has to be published as soon as the change is merged to master

### CODE REVIEW

File: documentation/release-latest/docs/rules/standard.md [hunk 1]
```diff
@@ -59,22 +59,22 @@ Requires a blank line before any class or function declaration. No blank line is
 === "[:material-heart:](#) Ktlint"
 
     ```kotlin
-    const val FOO_1 = "foo1"
```
Comment (paul-dingemans) [inline documentation/release-latest/docs/rules/standard.md:62]: `FOO_1` is used intentionally. Constant value which are deeply immutable property should be written in screaming case according to `property-naming` rule. Of course it should have been consistent with the disallowed example.

File: documentation/snapshot/docs/rules/standard.md [hunk 2]
```diff
@@ -59,22 +59,22 @@ Requires a blank line before any class or function declaration. No blank line is
 === "[:material-heart:](#) Ktlint"
 
     ```kotlin
-    const val FOO_1 = "foo1"
+    const val foo1 = "foo1"
```
Comment (paul-dingemans) [inline documentation/snapshot/docs/rules/standard.md:62]: `FOO_1` is used intentionally. Constant value which are deeply immutable property should be written in screaming case according to `property-naming` rule. Of course it should have been consistent with the disallowed example.

**Review by paul-dingemans — COMMENTED** [review]
Tnx for fixing. I will make a slight alteration before merge.

**Comment by k-taro56** [comment]
I learned a lot! Thank you :)

**Pull request merged**